### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by DockToFuture**
**How to categorize this PR?**

/area networking
/kind bug

**What this PR does / why we need it**:
Rephrase correctly that azure does not support calico with IP in IP as overlay network.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
none
```

### CODE REVIEW

**Review by timuthy — COMMENTED** [review]
/lgtm

File: docs/proposals/03-networking-extensibility.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Gardener is an open-source project that provides a nested user model. Basically,
 
 For the first set of users, the choice of network plugin might not be so important, however, for the second class of users (i.e., Hosted) it is important to be able to customize networking based on their needs.
 
-Furthermore, Gardener provisions clusters on different cloud-providers with different networking requirements. For example, Azure does not support Calico Networking [1], this leads to the introduction of manual exceptions in static add-on charts which is error prone and can lead to failures during upgrades.
+Furthermore, Gardener provisions clusters on different cloud-providers with different networking requirements. For example, Azure does not support Calico overlay networking with IP in IP [1], this leads to the introduction of manual exceptions in static add-on charts which is error prone and can lead to failures during upgrades.
```
Comment (vpnachev) [inline docs/proposals/03-networking-extensibility.md:14]: Can please update also https://github.com/gardener/gardener/blob/master/docs/extensions/network.md.

**Review by vpnachev — CHANGES_REQUESTED** [review]

File: docs/proposals/03-networking-extensibility.md (hunk 1 above)
Comment (DockToFuture) [reply docs/proposals/03-networking-extensibility.md:14]: I've updated it as well!

**Review by vpnachev — APPROVED** [review]
/lgtm

**Pull request merged**